src/application.rs

```diff
@@ -137,20 +174,85 @@ impl Application {
                 self.model.borrow_mut().update_app_list(app_list);
             }
 
+            IpcMessage::ZedAgentStatus(status) => {
```

Comment:
I can't comment on code above, but there's IpcMessage::LedBlinkCounter(led) case which just prints that we have LedBlinkCounter, is that intended?

Reply:
if you want to keep variable you can just write _led to silent compiler warning